public/styles/main.css

```diff
@@ -26,6 +26,15 @@
   /* Spacing */
   --spacing-unit: 1.5rem;
 }
+@media all and (min-width: 35rem) {
```

Comment:
Try to add reduced motion!

 https://css-tricks.com/introduction-reduced-motion-media-query/

Reply:
Fixed it